### PR TITLE
added a TodoListItem and an inline style to check when the item is ch…

### PR DESCRIPTION
**Pull request by hangthinguyen**
TodolistIitem component

### CODE REVIEW

File: src/App/App.css [hunk 1]
```diff
@@ -1,3 +1,4 @@
 .app {
-    max-width: 420px;
+    margin: 200px 0px;
```
Comment (vinhvanvu) [inline src/App/App.css:2]: why change max width to width?

File: src/Components/TodoListItem/TodoListItem.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faRulerHorizontal } from '@fortawesome/free-solid-svg-icons';
```
Comment (vinhvanvu) [inline src/Components/TodoListItem/TodoListItem.jsx:2]: unused import

File: src/index.css [hunk 3]
```diff
@@ -1,12 +1,16 @@
 body {
-  margin: 0;
+  margin: auto;
+  display: flex;
```
Comment (vinhvanvu) [inline src/index.css:3]: why add flexbox on body

File: src/App/App.css (hunk 1 above)
Comment (hangthinguyen) [reply src/App/App.css:2]: Because when I left it as max-width and tried to display flexbox on it, the width shrank.

File: src/index.css (hunk 3 above)
Comment (hangthinguyen) [reply src/index.css:3]: to center the app in the middle of the body

Comment (vinhvanvu) [reply src/index.css:3]: you should move this to .app instead since .app is the root of your react application.

File: src/App/App.css (hunk 1 above)
Comment (vinhvanvu) [reply src/App/App.css:2]: see other comment on body css

**Review by vinhvanvu — APPROVED** [review]

**Review by vinhvanvu — APPROVED** [review]

**Pull request merged**